File: src/main/java/org/jdc/template/model/database/main/individual/Individual.java
/*
 * Individual.java
 *
 * Created: 10/02/2012 01:40:29
 */



package org.jdc.template.model.database.main.individual;

public class Individual extends IndividualBaseRecord {
    public String getFullName() {
        return getFirstName() + " " + getLastName();
    }
}

File: src/main/kotlin/org/jdc/template/model/database/main/individual/Individual.kt
/*
 * Individual.kt
 *
 * Created: 02/14/2016 07:02:10
 */



package org.jdc.template.model.database.main.individual


class Individual : IndividualBaseRecord() {

    fun getFullName(): String {
        return firstName + " " + lastName
    }
}